src/EnergyPlus/WindowManagerExteriorOptical.cc

```diff
@@ -272,6 +273,22 @@ namespace WindowManager {
         }         // End of surface loop
     }
 
+    double GetSolarTransDirectHemispherical(EnergyPlusData &state, int ConstrNum)
```

Comment:
Again, are these `double` return types to ensure future compatibility with WCE in case we ever change to some other floating point type?